CalibMuon/DTCalibration/python/dtDQMClientAlca_cfg.py

```diff
@@ -10,7 +10,7 @@ class config: pass
 process.load("FWCore.MessageLogger.MessageLogger_cfi")
 
 process.load("Configuration.StandardSequences.Geometry_cff")
```

Comment:
you might want to consider `GeometryDB_cff`. `Geometry_cff` is run1 only geometry, see [this talk](https://indico.cern.ch/event/368326/contribution/1/material/slides/0.pdf)